sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -554,9 +565,11 @@ export class SessionReceiver {
    * @returns {Promise<void>}
    */
   async close(): Promise<void> {
+    this._isClosed = true;
     try {
       if (this._messageSession) {
         await this._messageSession.close();
+        this._messageSession.isReceivingMessages = false;
```

Comment:
At this point the underlying AMQP receiver link held by `this._messageSession` is closed.
I'd rather do `this_messageSession = undefined` rather than set `this._messageSession.isReceivingMessages = false`. 2 reasons:
- After this point the messageSession is useless, it cannot be used for anything
- This follows the same model we use for the normal streaming and batching receiver i.e remove any references (this._context.streamingReceiver and this._context.batchingReceiver) to it once it is closed